### ISSUE TITLE
Consider `pyogrio` engine for geopandas speedup

### ISSUE THREAD

**Issue by mfisher87** [PermafrostDiscoveryGateway/viz-3dtiles#26]
This will be the default in geopandas 1.0, but for now we can use it optionally: https://geopandas.org/en/stable/docs/reference/api/geopandas.read_file.html (see the "Note" callout)

**Comment by robyngit** [comment]
see also: https://github.com/PermafrostDiscoveryGateway/viz-workflow/issues/4 :)

**Comment by mfisher87** [comment]
Nice!

**Comment by robyngit** [comment]
I don't think so, this issue could cover using `pyogrio` in 3d tiles specifically
